### PR TITLE
Show delete button on single filter in Recordings

### PR DESCRIPTION
**Pull request by samwinslow**
A global rule to hide the delete button when only 1 action filter row is visible was breaking the filter experience on Recordings.

## Changes

Before: no way to reset/remove filter

Screen Shot 2021-11-15 at 11 30 47 AM

After:

Screen Shot 2021-11-15 at 11 31 08 AM

## How did you test this code?

Manual usability testing

### CODE REVIEW

**Review by rcmarron — APPROVED** [review]
LGTM

File: frontend/src/scenes/session-recordings/player/eventsListLogic.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { colonDelimitedDuration } from 'lib/utils'
 import { CellMeasurerCache } from 'react-virtualized/dist/commonjs/CellMeasurer'
 
 export const eventsListLogic = kea<eventsListLogicType>({
+    path: ['scenes', 'session-recordings', 'player', 'eventsListLogic'],
```
Comment (alexkim205) [inline frontend/src/scenes/session-recordings/player/eventsListLogic.ts:9]: Hm not quite sure what the context is here. Was this auto-generated by Kea?

File: frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx [hunk 2]
```diff
@@ -298,7 +298,6 @@ export function ActionFilterRow({
 
     const deleteButton = (
         <Button
-            style={filterCount === 1 ? { display: 'none' } : {}}
```
Comment (alexkim205) [inline frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx:301]: thats so weird that we had this global filter toggle. Nice find!

File: frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
 export function FunnelTab(): JSX.Element {
     const { insightProps } = useValues(insightLogic)
     const { loadResults } = useActions(insightLogic)
-    const { isStepsEmpty, filters, clickhouseFeaturesEnabled, aggregationTargetLabel } = useValues(
+    const { isStepsEmpty, filters, clickhouseFeaturesEnabled, aggregationTargetLabel, steps } = useValues(
```
Comment (alexkim205) [inline frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx:33]: I think `selectors.steps` is a selector that's dependent on the results we get after hitting the api. In the case where results are empty but filters.length === 1, the delete button will still show. I think what we want here is `selectors.filterSteps` or something similar

**Review by alexkim205 — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/session-recordings/player/eventsListLogic.ts (hunk 1 above)
Comment (samwinslow) [reply frontend/src/scenes/session-recordings/player/eventsListLogic.ts:9]: Yep! Kea-typegen added this change. It's related to the change to esbuild.

File: frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx (hunk 3 above)
Comment (samwinslow) [reply frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx]: Good catch, thanks! Just updated.

**Review by alexkim205 — APPROVED** [review]
nice!

**Pull request merged**